diesel_cli/src/infer_schema_internals/inference.rs

```diff
@@ -23,6 +23,38 @@ fn is_reserved(name: &str) -> bool {
         )
 }
 
+fn contains_unmappable_chars(name: &str) -> bool {
+    // Rust identifier names are restricted to [a-zA-Z0-9_].
+    !name.chars().all(|c| c.is_ascii_alphanumeric() || c == '_')
```

Comment:
I'm probably micro optimizing this, but wouldn't be 
```suggestion
    name.chars().any(|c| !(c.is_ascii_alphanumeric() || c == '_'))
```

more performant, as it does potential do an early return in the case of an invalid identifier?

Reply:
I believe `Iterator::all()` is also short-circuiting, so it should return early on the first invalid character as well.

Reply:
Oh right, that's true. (Not sure why I missed that 🤦‍♂ )